src/uu/shuf/src/shuf.rs

```diff
@@ -120,7 +120,6 @@ pub fn uu_app() -> Command {
         .version(crate_version!())
         .override_usage(format_usage(USAGE))
         .infer_long_args(true)
-        .args_override_self(true)
```

Comment:
This is correct for almost all arguments, but not for `-r` and `-z`, so to avoid regressing those, we should add `overrides_with` to those arguments. A test for those cases would be great too :)

Reply:
Oh, right, absolutely!

Story how this happened: I saw `ArgAction::SetTrue` and blindly assumed that this would behave identically to python's argparse, which accepts repeated arguments. I did attempt to check this assumption, but apparently forgot to test it with `-r` and `-z`. TIL.

Will be a new commit in the next push.